lms/static/scripts/ui-playground/components/ToolbarPage.tsx

```diff
@@ -125,15 +125,23 @@ export default function ToolbarPage() {
                               title="previous student"
                               variant="dark"
                             />
-                            <Select
-                              aria-label="Select student"
-                              classes="xl:w-80"
-                              id="student-selector"
+                            <label
```

Comment:
Now that we actually have this UI implemented, I'm not sure there is value in keeping the toolbar page, which is going to get out of sync with the actual implementation. We could decide to remove these prototypes. We'll always have the code in Git if we need to get them back.

Reply:
Sounds reasonable.